include/boost/geometry/algorithms/detail/disjoint/segment_box.hpp

```diff
@@ -64,10 +65,38 @@ struct disjoint_segment_box_sphere_or_spheroid
 
 public:
 
+    struct disjoint_info
+    {
+        enum type
+        {
+            intersect,
+            disjoint_no_vertex,
+            disjoint_vertex
+        };
+        disjoint_info(type t) : t_(t){}
+        operator type () const {return t_;}
+        type t_;
```

Comment:
Typically we put `m_` before member variable.

Reply:
I saw that you did this change in merge commit. Not the best way but ok. Btw my point was not to name it `m_` but `m_t` but ok this can be done later.